tests/js/t8404.nim

```diff
@@ -0,0 +1,29 @@
+discard """
+  targets: "c cpp js"
```

Comment:
move to some other dir since targets isnt' just js

Reply:
it's mainly for js target, for c cpp may never occurs that issue.

Reply:
still, tests/js is meant just for js; you could move this test to `tests/misc/tconv.nim` to simply to `tests/misc/t8404.nim`

(and keep `targets: "c cpp js"` which is good)